locale/de/settings.cfg

```diff
@@ -1,37 +1,37 @@
 [mod-setting-name]
-ltn-interface-console-level=Nachrichten Detailgrad
-ltn-interface-message-filter-age=Nachrichtenfilterdauer (ticks)
-ltn-interface-debug-logfile=Schreibe Debug Logdatei
+ltn-interface-console-level=Nachrichtendetailgrad
+ltn-interface-message-filter-age=Nachrichtenfilterdauer (Ticks)
+ltn-interface-debug-logfile=Debug-Protokollierung aktivieren
 
 ltn-dispatcher-enabled=Disponent aktiv
-ltn-dispatcher-nth_tick=Update Frequenz (ticks)
+ltn-dispatcher-nth_tick=Update-Intervall (Ticks)
 ltn-dispatcher-updates-per-tick=Updates pro Tick
 ltn-dispatcher-requester-threshold=Mindestmenge für Anforderung
-ltn-dispatcher-provider-threshold=Mindestmenge für Anbieter
-ltn-dispatcher-schedule-circuit-control=Fahrplan Schaltungsbedinungen
-ltn-dispatcher-depot-inactivity(s)=Depot Inaktivität (sek)
-ltn-dispatcher-stop-timeout(s)=Haltestellen Time-Out (sek)
-ltn-dispatcher-delivery-timeout(s)=Lieferzeit Time-Out (sek)
-ltn-dispatcher-requester-delivery-reset=Fahrplan bei Anforderer zurücksetzen
+ltn-dispatcher-provider-threshold=Mindestmenge für Angebot
+ltn-dispatcher-schedule-circuit-control=Fahrplanschaltungsbedinungen
+ltn-dispatcher-depot-inactivity(s)=Depotinaktivität (Sek.)
+ltn-dispatcher-stop-timeout(s)=Haltestellenunterbrechung (Sek.)

```

Comment:
Klingt nach google translate
stop-timeout ist die Zeit nach der ein Zug die Station verlassen muß auch wenn er nicht fertig be- entladen ist.

sinngemäße Übersetzung: "maximale Ladedauer"